### PR TITLE
✅ Re-enable tooltip interactive tests

### PR DESCRIPTION
**Pull request by Enriqe**
The tests were failing because 
1. We weren't waiting enough time for the animations to finish before checking taking the snapshot, and
2. At some point I mistakenly edited the file path in the config for a non-existent one. (d'oh!)

I re-ran the tests multiple times to make sure they weren't flaking.

### CODE REVIEW

File: examples/visual-tests/amp-story/amp-story-tooltip-desktop.js [hunk 1]
```diff
@@ -20,34 +20,40 @@ const {verifySelectorsVisible} = require('../../../build-system/tasks/visual-dif
 module.exports = {
   'tapping on a clickable anchor should show the tooltip': async (page, name) => {
     await page.tap('.next-container > button.i-amphtml-story-button-move');
-    await page.waitFor('amp-story-page#page-2[active]');
```
Comment (newmuis) [inline examples/visual-tests/amp-story/amp-story-tooltip-desktop.js]: Couldn't this still be flaky, if the navigation takes a long time?

I feel like maybe we should wait for page 2 to become active, _then_ wait however many milliseconds are necessary for the animation to complete?

**Comment by gmajoulet** [comment]
Percy diff LGTM

File: examples/visual-tests/amp-story/amp-story-tooltip-desktop.js (hunk 1 above)
Comment (Enriqe) [reply examples/visual-tests/amp-story/amp-story-tooltip-desktop.js]: Sure, I re-added those.

**Comment by newmuis** [comment]
nit: can you leave a comment `// for animations to finish` after each of the timed `waitFor` calls?

**Comment by Enriqe** [comment]
percy diffs ready for review and approval :)

**Comment by newmuis** [comment]
Approved

**Pull request merged**